source/common/runtime/runtime_features.h

```diff
@@ -35,5 +37,19 @@ class RuntimeFeatures {
 
 using RuntimeFeaturesDefaults = ConstSingleton<RuntimeFeatures>;
 
+// Helper class for runtime-derived boolean feature flags.
+class FeatureFlag {
+public:
+  FeatureFlag(std::string runtime_key, bool default_value, Runtime::Loader& runtime)
```

Comment:
Can this just take a `RuntimeFeatureFlag` as a parameter? Perhaps if nullptr the feature flag will always return true or something like that?

Reply:
I wanted to avoid adding a dependency on a proto, but it looks like this already depends on some v2 api protos. Changing it over.